example.exs

```diff
@@ -10,27 +10,35 @@ defmodule MP3Encoder.Pipeline do
   use Membrane.Pipeline
 
   alias Membrane.FFmpeg.SWResample.Converter
-  alias Membrane.Caps.Audio.Raw
 
   @impl true
   def handle_init(_ctx, filename) do
-    children = [
-      portaudio: Membrane.PortAudio.Source,
-      converter: %Converter{
-        input_caps: %Raw{channels: 2, format: :s16le, sample_rate: 48_000},
-        output_caps: %Raw{channels: 2, format: :s32le, sample_rate: 44_100}
-      },
-      encoder: Membrane.MP3.Lame.Encoder,
-      file: %Membrane.File.Sink{location: filename}
-    ]
+    spec =
+      child(:portaudio, %Membrane.PortAudio.Source{
+        channels: 2,
+        sample_format: :s16le,
+        sample_rate: 48_000
+      })
+      |> child(:converter, %Converter{
+        input_stream_format: %Membrane.RawAudio{
+          channels: 2,
+          sample_format: :s16le,
+          sample_rate: 48_000
+        },
```

Comment:
this shouldn't be necessary ;)

```suggestion
```

Reply:
Hi, yes you are right ; indeed this is not necessary. I have removed it now, thanks :slightly_smiling_face: